tests/functional/spec/answer_option_based_on_first_item_in_list.js

```diff
@@ -0,0 +1,59 @@
+const ListCollectorPage = require("../generated_pages/answer_based_on_first_item_in_list/list-collector.page.js");
+const ListCollectorAddPage = require("../generated_pages/answer_based_on_first_item_in_list/list-collector-add.page.js");
+const ListStatusQuestion = require("../generated_pages/answer_based_on_first_item_in_list/list-status.page.js");
+const FavouriteDrinkQuestion = require("../generated_pages/answer_based_on_first_item_in_list/favourite-drink.page.js");
+const ListStatusTwoQuestion = require("../generated_pages/answer_based_on_first_item_in_list/list-status-2.page.js");
+const SummaryPage = require("../generated_pages/answer_based_on_first_item_in_list/personal-details-section-summary.page.js");
+
+const HubPage = require("../base_pages/hub.page.js");
+
+describe("Answer Option Based on First Item in List", function() {
+  it("Given I am the first person in the list, When I get to the question page, Then I should see the default answer option", function() {
+    browser.openQuestionnaire("test_answer_based_on_first_item_in_list.json");
+    $(HubPage.submit()).click();
+    $(ListCollectorPage.yes()).click();
+    $(ListCollectorPage.submit()).click();
+    $(ListCollectorAddPage.firstName()).setValue("Marcus");
+    $(ListCollectorAddPage.lastName()).setValue("Twin");
+    $(ListCollectorAddPage.submit()).click();
+    $(ListCollectorPage.no()).click();
+    $(ListCollectorPage.submit()).click();
+    $(HubPage.submit()).click();
+    $(ListStatusQuestion.yes()).click();
+    $(ListStatusQuestion.submit()).click();
+    $(FavouriteDrinkQuestion.answer()).setValue("Orange Juice");
+    $(FavouriteDrinkQuestion.submit()).click();
+    expect($(ListStatusTwoQuestion.listStatus2TeaLabel()).getText()).to.contain("Tea");
+  });
```

Comment:
We should use multiple `describe` to split these up to make it easier to follow. This will make it a nicer output in the terminal. Adding comments also make it easier to follow.

Since both describes also open the schema, you can make use of the `beforeEach` handler.

Eg:
```js
describe('Placeholder Based on First Item in List"', () => {
  beforeEach(("Open questionnaire") => {
    browser.openQuestionnaire('test_placeholder_based_on_first_item_in_list.json');
  });

  describe('Given I am the first person in the list', () => {
    it('When I get to the question page, Then I should see the default answer option', () => {
      // Given
	  $(HubPage.submit()).click();
      $(ListCollectorPage.yes()).click();
      $(ListCollectorPage.submit()).click();
      $(ListCollectorAddPage.firstName()).setValue("Marcus");
      $(ListCollectorAddPage.lastName()).setValue("Twin");
      $(ListCollectorAddPage.submit()).click();
      $(ListCollectorPage.no()).click();
      $(ListCollectorPage.submit()).click();
      
	  // When
	  $(HubPage.submit()).click();
      $(ListStatusQuestion.yes()).click();
      $(ListStatusQuestion.submit()).click();
      $(FavouriteDrinkQuestion.answer()).setValue("Orange Juice");
      $(FavouriteDrinkQuestion.submit()).click();

	  // Then
      expect($(ListStatusTwoQuestion.listStatus2TeaLabel()).getText()).to.contain("Tea");
    });
  });

  describe('...', () => {
    it('...', () => {
      ...
    });
  });

});
```